IFC4x3/Properties/m/MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MinimumOperatingTemperature_1D6hoqiFP87eVLqmorvPDX" Name="MinimumOperatingTemperature" UniqueId="4d1abcb4-b0f6-481e-87d5-d30cb5e59361" PrimaryDataType="IfcThermodynamicTemperatureMeasure" />
```

Comment:
if you use `OperationalTemperatureRange_2hvjQhIJj5c9PrmgrOQl1i` above as suggested you do not need this as the lower bound of that property is this value.